src/main/java/homework3/ex2/page/Ex2DifferentElementsPage.java

```diff
@@ -0,0 +1,85 @@
+package homework3.ex2.page;
+
+import homework3.BasePage;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+import java.util.List;
+
+public class Ex2DifferentElementsPage extends BasePage {
+
+    @FindBy(css = "label:nth-child(1) > input[type=checkbox]")
+    private WebElement waterCheckbox;
+    @FindBy(css = "label:nth-child(3) > input[type=checkbox]")
+    private WebElement windCheckbox;
+    @FindBy(css = "label:nth-child(4) > input[type=radio]")
+    private WebElement selenRadio;
+    @FindBy(className = "colors")
+    private WebElement colorMenu;
+    @FindBy(css = ".colors > select > option:nth-child(4)")
```

Comment:
can be improverd with xpath: //option[contains(.,'Yellow')]

But I'd like you to make method:
selectColor(String color){
   // logic  
}